ci-operator/jobs/quarkus-qe/quarkus-openshift-interop/quarkus-qe-quarkus-openshift-interop-main-periodics.yaml

```diff
@@ -1,22 +1,22 @@
 periodics:
 - agent: kubernetes
-  cluster: build03
+  cluster: build05
```

Comment:
This is typically because of make update. I think even if you change it manually, then run `make update`, the command will still change it to whatever it wants.